server/db/schema.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE IF EXISTS pet_types CASCADE;
+DROP TABLE IF EXISTS adoptable_pets;
+
+CREATE TABLE pet_types (
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(255) NOT NULL,
+    img_url VARCHAR(255) NOT NULL,
+    description TEXT
+);
+
+CREATE TABLE adoptable_pets (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    img_url VARCHAR(255) NOT NULL,
+    age INTEGER,
+    vaccination_status BOOLEAN NOT NULL,
+    adoption_story TEXT NOT NULL,
+    available_for_adoption BOOLEAN NOT NULL,
+    pet_type_id INTEGER REFERENCES pet_types
+);
```

Comment:
We can first focus on just the `pet_types`. any other tables should not be included in this PR.  We can just remove the second CREATE TABLE command